source/includes/_invoice.md

```diff
@@ -2169,8 +2169,6 @@ Delete a Credit Balance Adjust (`CBA_ADJ`) invoice item. There are some limitati
 1. Deleting a positive `CBA_ADJ` (credit generation), may lead the system to reclaim portion of the used credit, possibly leaving some invoices with a balance. Example:
 
 Given an invoice, I1,  where user added some credit ($12), we would see the following items: {`CREDIT_ADJ`: -12, `CBA_ADJ`: +12}. Given another invoice, I2, where the system invoiced for a recurring subscription, and where some of this credit was consumed, we would see the following items: {`RECURRING`: +10, `CBA_ADJ`: -10}. Deleting the `CBA_ADJ` from I1, would lead to the following resulting invoices:  I1 {`CREDIT_ADJ`: 0, `CBA_ADJ`: 0} and I2 {`RECURRING`: +10, `CBA_ADJ`: 0}. The system zeroed-out the credit generation and the part that was used, and as a result I2 would be left with a balance of +10.
-
-
 2. System generated credit
```

Comment:
Numbering restarts at 1 here so removed this space to see if this causes the numbering to continue from `2`

Reply:
ack -- maybe it's because of the space above (line 2170)?